earth_enterprise/src/common/packet.cpp

```diff
@@ -78,14 +79,14 @@ etPointerTranslator * etDrawablePacket::packetPointerTranslator = NULL;
 int etPattern::hash(const char* pattern)
 {
   const int M = 26;
```

Comment:
also unrelated, but do we know what this is?

would it be worthwhile to make primes a std::array?